modules/bullet/collision_object_bullet.cpp

```diff
@@ -40,12 +40,16 @@
 #define enableDynamicAabbTree true
 #define initialChildCapacity 1
 
+CollisionObjectBullet::ShapeWrapper::~ShapeWrapper() {
+	bulletdelete(bt_shape)
```

Comment:
It looks like this broke compound shapes.  If I load a scene with a rigidbody and a single collision shape, add another shape to the rigidbody (thus creating a compound shape), and save the scene, I get the following error messages and a crash:
```
pure virtual method called
terminate called without an active exception
```

stack trace:
![image](https://user-images.githubusercontent.com/2694663/29744274-8f1db0d8-8a56-11e7-8e0e-f49ea4840e82.png)

It looks like saving the scene results in a queue_free() (no clue why),  which calls the ShapeWrapper destructor at some point (which frees the bullet shape), and eventually calls the virtual method shape->getMargin(), which fails because the bullet object was destroyed.

I can also reproduce this by calling queue_delete on a rigidbody with a compound shape when the game is running.

Removing "bulletdelete" fixes the problem, but that feels wrong as you do want to clean it up.  Maybe the fix would be to ensure shape->active is false after queue_free() is called?